addons/binding/org.openhab.binding.solarlog/src/main/java/org/openhab/binding/solarlog/SolarLogBindingConstants.java

```diff
@@ -0,0 +1,85 @@
+/**
+ * Copyright (c) 2014-2016 by the respective copyright holders.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.solarlog;
+
+import org.eclipse.smarthome.core.thing.ThingTypeUID;
+
+/**
+ * The {@link SolarLogBinding} class defines common constants, which are
+ * used across the whole binding.
+ *
+ * @author Johann Richard - Initial contribution
+ */
+public class SolarLogBindingConstants {
+
+    public static final String BINDING_ID = "solarlog";
+
+    // List of all Thing Type UIDs
+    public final static ThingTypeUID THING_SOLARLOG = new ThingTypeUID(BINDING_ID, "solarlog_js");
```

Comment:
This creates the identifier "solarlog:solarlog_js". Please remove the "solarlog" here. Maybe use the long version of "js" as an identifier, whatever js stands for though :)